capitalize_decorator.rb

```diff
@@ -0,0 +1,11 @@
+# For the CapitalizeDecorator:
+# Create a class that inherits from the base Decorator class.
+# Implement a method correct_name that capitalizes the output of @nameable.correct_name.
+
+require './base_decorator'
+
+class CapitalizeDecorator < Base_Decorator
+  def correct_name
+    @nameable.correct_name.capitalizes
+  end
```

Comment:
- [ ]  You should fix the typo error in your method because `capitalizes `should be `capitalize `to work properly, this will make your method work properly to capitalize the name as it is supposed to be.